what4/src/What4/Protocol/SMTLib2/Syntax.hs

```diff
@@ -184,7 +185,11 @@ qf_bv = Logic "QF_BV"
 -- | Set the logic to all supported logics.
 allSupported :: Logic
 allSupported = Logic "ALL_SUPPORTED"
+{-# DEPRECATED allSupported "Use allLogic instead" #-}
```

Comment:
It would be worth noting in `what4/CHANGES.md` that `allSupported` is now deprecated in favor `allLogic`.